Plugins/SmokeFrameworkGenerateClient/plugin.swift

```diff
@@ -53,4 +99,82 @@ struct SmokeFrameworkGenerateClientPlugin: BuildToolPlugin {
         
         return [command]
     }
+    
+    private func getModelFilePathOverride(target: Target, config: SmokeFrameworkCodeGen,
+                                          baseFilePath: PackagePlugin.Path) throws -> String {
+        // find the model for the current target
+        let filteredModelLocations = config.modelLocations?.targetMap.compactMap { (targetName, modelLocation) -> ModelLocation? in
```

Comment:
nit: Can this be `let targerModelLocationOptional = config.modelLocations?.targetMap[target.name]`?

Reply:
Oh yeah, it is a dictionary. Fixed.